libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -198,14 +204,21 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
 
     /** @hidden */
     private _getDialogOffset(): number {
-        const dialogBody = this._elRef.nativeElement.parentElement;
         let retVal = 0;
-        if (dialogBody.tagName.toLowerCase() === 'fd-dialog-body') {
-            if (dialogBody.querySelector('.' + 'fd-title--h2')) {
-                retVal = dialogBody.querySelector('.' + 'fd-title--h2').offsetHeight;
+        if (this._dialogBodyComponent) {
```

Comment:
I would do it like this:
`this._dialogBodyComponent.addClass('fd-dialog-body--no-vertical-padding')`
As for height, resizeObserver might help you out with window resize and dynamic content

Reply:
this is actually a property of dialogConfig so I will use that